tracee-ebpf/tracee/events_processor.go

```diff
@@ -255,6 +261,39 @@ func (t *Tracee) processEvent(event *external.Event) error {
 			return fmt.Errorf("error parsing cgroup_rmdir args: %v", err)
 		}
 		t.containers.CgroupRemove(cgroupId)
+
+	case DetectHookedSyscallsEventID:
+		doOnce.Do(func() {
```

Comment:
Why do we want to guard this with sync.Once?
What if we will want to trigger the event in runtime in the future?

Reply:
Because now we only check this event when tracee start.
if in the future we will want to call this event in runtime we need to make some changes that can in clude that removing that guard, for example we will need to change the mechaniqe of the `ioctl trigger event` to something more dynamic (now its sonly triggers it when tracee starts).
if you still think it isn't necessary , ill remove it (: